### DISCUSSION TITLE
How to access the videos list?

### DISCUSSION THREAD

**Discussion by Miala-python** [JunkFood02/Seal#1352] [Q&A] [answered]
Hello, I want to create a YouTube alternative player and I was wondering :
What method do you use to access the list of videos from the playlist URL ?

**Comment by JunkFood02** [comment, marked as answer]
https://github.com/JunkFood02/Seal/blob/c6f271ee8c3a457c82072d3ce892d04e647cf336/app/src/main/java/com/junkfood/seal/util/DownloadUtil.kt#L90
